### DISCUSSION TITLE
Change the license of OvenMediaEngine from GPLv3 to AGPLv3

### DISCUSSION THREAD

**Discussion by getroot** [OvenMediaLabs/OvenMediaEngine#1069] [Announcement]
We have decided to change the license of OvenMediaEngine from GPLv3 to AGPLv3.

Most of the people in the OvenMediaEngine community share their valuable experiences. And it has accumulated and the OvenMediaEngine community has become a collective intelligence. Through it, we continue to evolve together. I'm very proud to be one of the open-source developers and creators of OvenMediaEngine. And thanks to all contributors and users.

However, we found that some users monopolize their experiences and the modified OvenMediaEngine source code. And we sometimes compete with the OvenMediaEngine forks they made. I'm someone who believes that competition drives development, but this can't be fair competition because we disclose all information and they don't. It destroys the open source community and undermines our motivation to operate as open source.

In order for us to keep OvenMediaEngine open source, we must protect our investment. So we had a lot of discussions over a long period then the decision we made was to change the license of OvenMediaEngine to AGPLv3.

Please note that the new license is applied from the next release of 0.12.11. If you want to use the existing license, GPLv3, use version 0.12.11 or lower.

For more information about AGPLv3, please refer to this URL: https://www.gnu.org/licenses/why-affero-gpl.html

The key point is - "if you run a modified program on a server and let other users communicate with it there, your server must also allow them to download the source code corresponding to the modified version running there."

If you require a license other than AGPLv3, please email us at contact@airensoft.com

Thank you.

**Comment by banagale** [comment]
I'm sure this must have been a difficult decision, but it was the right one when facing problems like what you've described. Thank you for all your hard work on this software.
